src/GovUk.Education.ExploreEducationStatistics.Data.Model/LocalAuthorityDistrict.cs

```diff
@@ -1,41 +1,18 @@
 namespace GovUk.Education.ExploreEducationStatistics.Data.Model
 {
-    public class LocalAuthorityDistrict: IObservationalUnit
+    public class LocalAuthorityDistrict : ObservationalUnit
     {
-        public string Code { get; set; }
-        public string Name { get; set; }
-
-        private LocalAuthorityDistrict()
+        public LocalAuthorityDistrict()
         {
         }
-
-        public LocalAuthorityDistrict(string code, string name)
+        
+        public LocalAuthorityDistrict(string code, string name) : base(code, name)
         {
-            Code = code;
-            Name = name;
         }
-
+        
```

Comment:
This whitespace creeping in made me look into Rider settings for formatting during saving. I was sure Rider had a set of On-Save actions where we could turn on things like formatting but I can't see them now.

Reply:
Edit: I see a feature request, looks like it hasn't been implemented 😭

Reply:
I'm gob-smacked!  Do you manually format them as you go?

Reply:
Ha ha, I use a combination of keyboard shortcuts to reformat code and organise imports. I know these can both be automated in the commit options. I think I've tried these in the past and didn't get on with them. I'll try them out again.